__tests__/app.test.js

```diff
@@ -28,3 +29,41 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("GET description of all endpoints", () => {
```

Comment:
Instead of these two tests, you could be more specific and require in the endpoints.json here, and test your response looks the same as the endpoints json (the readfile is not required)